### PR TITLE
Add `verdict` guidance to `LLM query`

### PR DESCRIPTION
**Pull request by Nandini-Jaiswal**
# Description

This PR adds `verdicts` to `LLM Query` to tune the LLM response.

# Checklist:

- [x] The code follows the [coding guidelines](https://github.com/omegaup/omegaup/blob/main/frontend/www/docs/Coding-guidelines.md) of omegaUp.
- [x] The tests were executed and all of them passed.
- [x] If you are creating a feature, the new tests were added.
- [x] If the change is large (> 200 lines), this PR was split into various Pull Requests. It's preferred to create one PR for changes in controllers + unit tests in PHPUnit,  and then another Pull Request for UI + tests in Jest, Cypress or both.

### CODE REVIEW

File: stuff/teaching_assistant/teaching_assistant.py [hunk 1]
```diff
@@ -365,19 +394,79 @@ def conjure_query(  # pylint: disable=R0913
...
+    verdict_guidance = {
+        "WA": (
+            "The submission got Wrong Answer (WA). Extract the logic "
+            "from the code and see what's wrong. "
+            "Focus on algorithmic correctness and edge cases."
+        ),
+        "PA": (
+            "The submission got Partial Acceptance (PA). "
+            "What optimizations can be done to pass all the test cases? "
+            "Look for efficiency improvements."
```
Comment (heduenas) [inline stuff/teaching_assistant/teaching_assistant.py:415]: PA is also given if the algorithm is only partially correct

Comment (heduenas) [reply stuff/teaching_assistant/teaching_assistant.py:415]: @Nandini-Jaiswal can you include that in your code? Other than that the change looks good.

Comment (Nandini-Jaiswal) [reply stuff/teaching_assistant/teaching_assistant.py:415]: PTAL!

**Review by heduenas — APPROVED** [review]
